src/Provider/FreeGeoIp/Readme.md

```diff
@@ -10,6 +10,27 @@
 This is the Free GeoIp provider from the PHP Geocoder. This is a **READ ONLY** repository. See the
 [main repo](https://github.com/geocoder-php/Geocoder) for information and documentation. 
 
+# Freegeoip Shutdown
```

Comment:
I would change the titles "level": `##` (+1 level) here

Reply:
Updated (and the others)!